src/components/date-picker/z-date-picker/index.stories.ts

```diff
@@ -59,32 +60,42 @@ export const Date: Story = {
   args: {
     datePickerId: "picker-01",
     label: "ZDatePicker with date",
+    value: "2021-01-01",
```

Comment:
provando ho visto che il valore che viene settato è diverso da quello che viene passato qui. al primo render della story, la input ha `20-01-2024`.

inoltre mi sembra che non venga lanciato nessun evento quando cambia il valore passato alla prop, non so se è una cosa da attenzionare 🤔

Reply:
Per quanto riguardo evento sul cambio props io direi che è corretto così. Se si imposta da "fuori" il valore è corretto che non viene emesso l'evento "dateSelect", in quanto non è stata selezionata una data o non è cambiata la data dall'interno del componente